src/reducers/collapseContext.ts

```diff
@@ -1,15 +1,23 @@
-import { parentOf, pathToContext, reducerFlow, unroot } from '../util'
+import { head, parentOf, pathToContext, reducerFlow, unroot } from '../util'
 import { State } from '../util/initialState'
-import { getChildren } from '../selectors'
+import { getAllChildren, simplifyPath } from '../selectors'
 import { archiveThought, existingThoughtMove, setCursor } from '../reducers'
+import _ from 'lodash'
+import existingThoughtDelete from './existingThoughtDelete'
+
+interface Options {
+  preventCursorArchive?: boolean,
```

Comment:
Let's call this option `deleteCursor`. "prevent" suggests that it's prevented altogether.